test/createHistoryStream.js

```diff
@@ -126,7 +122,7 @@ test('limit', (t) => {
         pull(
           sbot.createHistoryStream({ id: keys.id }),
           pull.collect((err, results) => {
-            t.equal(results.length, 2)
+            t.equal(results.length, 3)
```

Comment:
Notice that this test is fundamentally testing something different now. I think the previous was actually testing the wrong thing, and it kept on passing because this test **didn't** have an onDrain. I took a careful look at what the published messages were, and I think there are 3.

Reply:
Yes